spec/03-plugins/23-rate-limiting/04-access_spec.lua

```diff
@@ -202,6 +202,8 @@ for _, strategy in helpers.each_strategy() do
                 redis_server_name = redis_conf.redis_server_name,
                 redis_password    = REDIS_PASSWORD,
                 redis_database    = REDIS_DATABASE,
+                error_code        = 404,
```

Comment:
I think it would be good to add a distinct test case for this custom status code/message feature so that things are a little bit more clear.

Reply:
Done, I agree this is better. The goal was to keep the PR small.